trade-finance-manager-ui/templates/case/amendments/underwriting/amendment-banks-decision/amendment-edit-banks-decision.njk

```diff
@@ -62,12 +62,16 @@
             })
           }}
 
-          {{ govukButton({
-            text: "Continue",
-            attributes: {
-              'data-cy': 'submit-button'
-            }
-          }) }}
+          {% if userCanEdit %}
+
+            {{ govukButton({
+              text: "Continue",
+              attributes: {
+                'data-cy': 'submit-button'
```

Comment:
check the naming convention for the other files related to amendments. 
we tend to use `continue-amendment`. I've changed it in my PR though to `amendment--continue-button` so use this pattern moving forward